frontend/projects/ui/src/app/pages/marketplace-routes/marketplace-list/marketplace-list-content/marketplace-list-content.component.ts

```diff
@@ -22,9 +31,17 @@ export class MarketplaceListContentComponent {
   @Input()
   name = ''
 
+  @Input()
+  description: string | undefined = undefined
+
   category = 'featured'
   query = ''
 
+  ngOnInit(): void {
+    this.description = this.description
+      ? this.markdown.transform(this.description)
```

Comment:
could not get this working in html as a ` | markdown` since this component doesn't have a module

Reply:
voided